.github/workflows/installers-conda.yml

```
@@ -29,7 +29,7 @@ concurrency:
   group: installers-conda-${{ github.ref }}
   cancel-in-progress: true
 
-name: Create conda-based installers for Linux
+name: Conda-based installers
```

```suggestion
name: Nightly conda-based installers
```

Another suggestion that just occurred to me to make the associated badge more meaningful to users.

---

Is there a way to name the badge separate from the workflow name? I think this makes a lot of sense for the badge, but not so much for the workflow. Nevertheless, if there is no other way, then I'll change the name of the workflow.

---

> Is there a way to name the badge separate from the workflow name?

I don't think there is, i.e. the badge name is the same as the workflow one. So please apply this suggestion to merge this one.